src/main/build/debug.c

```diff
@@ -61,7 +61,7 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "SBUS",
     "FPORT",
     "RANGEFINDER",
-    "RANGEFINDER_QUALITY",
+    "RNGFNDR_QUALITY",
```

Comment:
I like `RNGFNDR` - just a jumble of consonants, but pretty clear what it is. :-D